src/libs/state.ts

```diff
@@ -270,6 +270,8 @@ export const workspaceStore = atom<InitializedWorkspaceWrapper | undefined>(unde
 
 export const workspacesStore = atom<WorkspaceWrapper[]>([]);
 
+export const cloningWorkspacesStore = atom<WorkspaceInfo[]>([]);
```

Comment:
Cloning workspaces require a separate store, so the polling can be consistent through navigation, and potentially without the workspaces store being initialized.

The store is only for `WorkspaceInfo` instead of the entire workspace, because that's what is returned by the `NewWorkspaceModal`.